pep-0705.rst

```diff
@@ -0,0 +1,96 @@
+PEP: 705
+Title: Recording provenance of installed packages
+Author: Fridolin Pokorny <fridolin.pokorny at gmail.com>, Trishank Karthik Kuppusamy <karthik@trishank.com>
+Sponsor:
+PEP-Delegate:
+Discussions-To: https://discuss.python.org/t/pep-705-recording-provenance-of-installed-packages/23340
+Status: Draft
+Type: Process
+Content-Type: text/x-rst
+Created: 30-Jan-2002
+Post-History:
+
+
+Abstract
+========
+
+This PEP describes a way to record provenance of Python packages installed.
+The record is created by an installer and is available to users in a form of a
+JSON file ``direct_url.json`` in ``.dist-info`` directory. The PEP is an
+extension to :pep:`610` for cases when installed packages are comming from a
+package index.
+
+
+Motivation
+==========
+
+Installing a Python package involves downloading the package from a source and
+extracting its content to an appropriate place. After the installation process
+is done, information about the artifact used as well as its source is generally
+lost. Nevertheless, there are use cases for keeping records of artifacts used
+for installing packages and their provenance.
+
+Python wheels can be built with different compiler flags or supporting
+different wheel tags.  In both cases, users might get into a situation in which
+multiple wheels might be considered by installers (possibly from different
+package indexes) and immediately finding out which artifact was actually used
+during the installation might be helpful. This way, tools reporting software
+installed, such as tools reporting SBOM (Software Bill of Material), might give
+more accurate reports.
+
+The motivation described in this PEP is a direct extension to :pep:`610`.
+Besides stating information about packages installed from an URL, installers
+SHOULD record information also for packages installed from Python package
+indexes or from filesystem.
+
+
+Examples
+========
```

Comment:
You've jumped right to the examples before introducing any actual specification (and I don't see a section titled such anywhere). I would suggest starting with a clear, precise and unambiguous specification first, which is a preliminary requirement to be able to accept your PEP.